src/changelogs/tags.py

```diff
@@ -0,0 +1,342 @@
+            'shops': 'Shop',
+            'shop': 'Shop',
+            'minimap': 'Map',
+            'minimaps': 'Map',
+            'Minimap': 'Map',
+            'parry': 'Parry',
+            'parried': 'Parry',
+            'light melee': 'Light Melee',
+            'heavy melee': 'Heavy Melee',
+            'Light melee': 'Light Melee',
+            'Heavy melee': 'Heavy Melee',
+            'melee': 'Melee',
+            'melees': 'Melee',
+            'pause': 'Pause',
+            'pauses': 'Pause',
+            'paused': 'Pause',
+            'vaults': "Sinner's Sacrifice",
+            'vault': "Sinner's Sacrifice",
+            'Vault': "Sinner's Sacrifice",
+            "Sinner's sacrifice": "Sinner's Sacrifice",
+            'Sinners sacrifice': "Sinner's Sacrifice",
+            "Sinners' sacrifice": "Sinner's Sacrifice",
+            'Sinner sacrifice': "Sinner's Sacrifice",
+            "sinner's sacrifice": "Sinner's Sacrifice",
+            'sinners sacrifice': "Sinner's Sacrifice",
+            "sinners' sacrifice": "Sinner's Sacrifice",
+            'sinner sacrifice': "Sinner's Sacrifice",
+            'flex slot': 'Flex Slot',
+            'Flex slot': 'Flex Slot',
+            'flex slots': 'Flex Slot',
+            'Flex': 'Flex Slot',
+            'guns': 'Gun',
+            'gun': 'Gun',
+            'Guns': 'Gun',
+            'Pits': 'Pit',
+            'pit': 'Pit',
+            'pits': 'Pit',
+        }
+
+        # Relations between a child and parent tag.
+        # i.e. Abrams is a parent to Siphon Life, and a child to Hero
+        # tags below are after _remap_tag() is called
+        # key = parent
+        # value = children
+        # also see self.parent_lookup
+        self.tag_tree = {
+            'Item': {'Weapon Item': {}, 'Vitality Item': {}, 'Spirit Item': {}},
+            'Hero': {'Ability': {}, 'Gun': {}},
+            'Melee': {
+                'Light Melee': {},
+                'Heavy Melee': {},
+                'Parry': {},
+            },
+            'NPC': {'Denizen': {}, 'Creep': {'Denizen': {}, 'Trooper': {}}, 'Mid-Boss': {}},
+            'Objective': {
+                'Guardian': {},
+                'Base Guardian': {},
+                'Walker': {},
+                'Patron': {'Weakened Patron': {}},
+                'Shrine': {},
+            },
+            'Souls': {'Soul Orb': {}, "Sinner's Sacrifice": {}, 'Soul Urn': {}},
+            'Breakable': {'Crate': {}, 'Golden Statue': {}},
+            'Shop': {},
+            'Map': {
+                'Department Store': {},
+                'Arcade': {},
+                'Fish Market': {},
+                'Basement': {},
+                'Bridge': {},
+                'Hotel': {},
+                'Pharmacy': {},
+                'Penn Station': {},
+                'Grand Central': {},
+                'Pit': {},
+            },
+            'Powerup': {},
+            'Zipline': {},
+            'Rejuvenator': {},
```

Comment:
Would rejuvenator make sense in `mid-boss`?

Reply:
I thought about this recently;

The idea is, whenever we find changelogs regarding Rejuvenator, do we want them to also end up on the Mid-Boss page. Or in other terms, when we visit the Mid-Boss#Update history section, do we want to see changes regarding Rejuvenator. Personally, I would say no. They may be in a similar mechanic but I'm not sure thats sufficient to nest Rejuv under Mid-Boss. 

The alternative I thought of was to instead nest both of these under some tag like "Mid-Boss Mechanics". But, I can't think of a better name and it seems a bit overkill. If you can think of a more appropriate name lmk though

Reply:
I totally see your point, I think its fine then. The wiki should allow for multiple filters, so you could in theory ask for both "Mid Boss" and "Rejuvinator" without the need of a parent tag